kerastuner/engine/tuner_utils.py

```diff
@@ -94,17 +98,65 @@ def on_epoch_end(self, epoch, logs=None):
 # TODO: Add more extensive display.
 class Display(object):
 
-    def __init__(self, verbose=1):
+    def __init__(self, oracle, verbose=1):
         self.verbose = verbose
+        self.oracle = oracle
 
     def on_trial_begin(self, trial):
         if self.verbose >= 1:
-            display.section('Starting new trial')
+            print()
+            trial_number = self.oracle.get_trial_number(trial)
+            total_trials = self.oracle.max_trials or '?'
+            print('Search: Running Trial {}/{}'.format(trial_number, total_trials))
+            print()
+
+            self.trial_start = time.time()
+
+            template = "{0:20}|{1:10}|{2:20}"
+            best_trials = self.oracle.get_best_trials()
+            if len(best_trials) > 0:
+                best_trial = best_trials[0]
+            else:
+                best_trial = None
+            print(template.format('Hyperparameter', 'Value', 'Best Value So Far'))
+            if trial.hyperparameters.values:
+                for hp, value in trial.hyperparameters.values.items():
+                    best_value = str(best_trial.hyperparameters.values.get(hp)) if best_trial else '?'
+                    print(template.format(hp, str(value), best_value))
+            else:
+                print('default configuration')
+            print()
 
     def on_trial_end(self, trial):
         if self.verbose >= 1:
-            display.section('Trial complete')
-            trial.summary()
+            if IS_NOTEBOOK:
+                display.clear_output()
+            else:
+                print() # Separate with a newline
+
+            trial_number = self.oracle.get_trial_number(trial)
+            total_trials = self.oracle.max_trials or '?'
+
+            time_taken_str = self.format_time(time.time() - self.trial_start)
+            print('Trial {}/{} Complete [{}]'.format(trial_number, total_trials, time_taken_str))
+
+            if trial.score is not None:
+                print('{}: {}'.format(self.oracle.objective.name, trial.score))
+
+            best_trials = self.oracle.get_best_trials()
+            if len(best_trials) > 0:
+                best_score = best_trials[0].score
+            else:
+                best_score = None
+            print('Best {} So Far: {}'.format(self.oracle.objective.name, best_score))
+
+            time_remaining = self.oracle.get_time_remaining()
```

Comment:
Let's just show the time the last trial took, and the total time since starting. Getting the total number of trials remaining from the chief Oracle adds too much complexity IMO